src/client/content_cao.cpp

```diff
@@ -1489,16 +1489,14 @@ bool GenericCAO::visualExpiryRequired(const ObjectProperties &new_) const
 	const ObjectProperties &old = m_prop;
 	// Ordered to compare primitive types before std::vectors
 	return old.backface_culling != new_.backface_culling ||
-		old.initial_sprite_basepos != new_.initial_sprite_basepos ||
```

Comment:
I think it would make sense to list those visual-related parameters above so that they're not re-added again in the future.
For example:
```
// Check for different parameters that are relevant to expireVisuals.
// Checking for updateTextures-relevant parameters is not wanted here.
```

Reply:
Added an explanatory comment.